contracts/Staking.sol

```diff
@@ -820,17 +824,34 @@ contract Staking is Pauser, Whitelist {
         epochIdx = _epochIdx;
         currentEpochIdx = _epochSigners;
```

Comment:
currentEpochIdx is not a good name (I forget why I use this name).  We may use currentEpochValidators instead :)